src/xenia/cpu/backend/x64/x64_sequences.cc

```diff
@@ -1708,6 +1724,17 @@ struct DIV_I8 : Sequence<DIV_I8, I<OPCODE_DIV, I8Op, I8Op, I8Op>> {
         } else {
           e.movsx(e.ax, i.src1);
         }
+        e.cmp(i.src2, minusOne);
+        e.jne(normalDiv);
+        e.cmp(e.al, minInt8);
```

Comment:
dividend = int_min is extremely unlikely while divisor = -1 is much more likely, so you can reverse the order of the checks (first check dividend) so it will skip the second check when dividend != int_min, leading to little reduced overall amount of checks.

Reply:
Good point